app/templates/main/uncaughtValidation.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block pageTitle %}Sorry, there is a problem with the service – {{ config['SERVICE_NAME'] }} – GOV.UK{% endblock pageTitle %}
+
+{% set mainClasses = "govuk-main-wrapper--l" %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
+        <p class="govuk-body">Try again later.</p>
+        <p class="govuk-body">Contact us at <a href="mailto: fsd.support@levellingup.gov.uk">fsd.support@levellingup.gov.uk</a>. Do not send your return or any attachments using the help email.</p>
```

Comment:
remove whitespace in mailto link
```suggestion
        <p class="govuk-body">Contact us at <a href="mailto:fsd.support@levellingup.gov.uk">fsd.support@levellingup.gov.uk</a>. Do not send your return or any attachments using the help email.</p>
```